Druid/CompiledMethod.extension.st

```diff
@@ -1,5 +1,13 @@
 Extension { #name : #CompiledMethod }
 
+{ #category : #'*Druid' }
+CompiledMethod >> hasPragmaNamed: pragmaName setting: argumentValue [
```

Comment:
I don't like this :)

This protocol to compare pragmas is too complicated.

First, it should be probably pushed to Pharo.
Second, France.
Third, I would simply expect to do:

```smalltalk
method hasPragma: (Pragma from #(name: 1)).
```

And properly define equality in pragmas so that two equal pragmas return true if they are the same.